pkg/activator/revision_test.go

```diff
@@ -62,7 +62,9 @@ func TestActiveEndpoint_Reserve_WaitsForReady(t *testing.T) {
 	select {
 	case <-ch:
 		t.Error("Unexpected result before revision is ready.")
-	case <-time.After(100 * time.Microsecond):
+	case <-time.After(1200 * time.Millisecond):
+		// Wait long enough, so that ActiveEndpoint Go routine sets up the Watch.
+		// It does fire any events internally, so we have to "sleep".
```

Comment:
I am not sure what is meant by this here. Can you clarify this further?

Also, I think you are missing a not?

```suggestion
		// It does not fire any events internally, so we have to "sleep".
```